report-items/graph/formatting-a-graph/title.md

```diff
@@ -1,34 +1,32 @@
 ---
 title: Title
-page_title: Graph Report Item Title 
-description: "Learn how to style and format the title of the Telerik Reporting Graph report item."
+page_title: Graph Report Item Title
+description: "Learn how to add, style and format the title of the Telerik Reporting Graph report item in the Report Desginers."
 slug: telerikreporting/designing-reports/report-structure/graph/formatting-a-graph/title
 tags: telerik, reporting, report, items, graph, styling, formatting, title
 published: True
 previous_url: /GraphFormattingTitle
 ---
 
-# Title
+# Title of the Graph
 
```

Comment:
How about "Graph Title" or "The Graph Title"? "Title of the graph" sounds kind of strange to me